### PR TITLE
Add notes for Chocolatey CLI 2.2.1

### PR DESCRIPTION
**Pull request by gep13**
## Description Of Changes

Add release notes for Chocolatey CLI v2.2.1.

## Motivation and Context

There is a release coming up, and we need to get this documented.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

**Comment by gep13** [comment]
@AdmiringWorm @pauby this is ready for review, however, please don't merge once the review is complete.  I will do this when everything else is ready.

File: input/en-us/choco/release-notes.md [hunk 1]
```diff
@@ -46,6 +46,22 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+## 2.2.1 (August 3, 2023)
+
+> :choco-warning: **WARNING**
+>
+> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 1.x versions to 2.x.
+
+### Bug Fixes
+
+- Fix - Can't find choco.exe after upgrading Chocolatey with MSI - see [#3286](https://github.com/chocolatey/choco/issues/3286).
+- Fix - Unable to do non-elevated operations when cache location is set to restricted directory - see [#3291](https://github.com/chocolatey/choco/issues/3291).
```
Comment (AdmiringWorm) [inline input/en-us/choco/release-notes.md:58]: I think

```suggestion
- Fix - Unable to do non-elevated operations when cache location is set to a restricted directory - see [#3291](https://github.com/chocolatey/choco/issues/3291).
```

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]
Other than a single comment, this look good to me

**Review by AdmiringWorm — DISMISSED** [review]
LGTM

File: input/en-us/choco/release-notes.md [hunk 2]
```diff
@@ -46,6 +46,22 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
 
+## 2.2.1 (August 3, 2023)
+
+> :choco-warning: **WARNING**
+>
+> Refer to our [Upgrade Guide](xref:upgrading-to-chocolatey-v2-v6) for recommendations before upgrading from 1.x versions to 2.x.
+
+### Bug Fixes
+
+- Fix - Can't find choco.exe after upgrading Chocolatey with MSI - see [#3286](https://github.com/chocolatey/choco/issues/3286).
```
Comment (pauby) [inline input/en-us/choco/release-notes.md:57]:
```suggestion
- Fix - Can't find `choco.exe` after upgrading Chocolatey CLI using the MSI - see [#3286](https://github.com/chocolatey/choco/issues/3286).
```

File: input/en-us/choco/release-notes.md [hunk 3]
```diff
@@ -46,6 +46,22 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
...
+- Fix - Unable to do non-elevated operations when cache location is set to a restricted directory - see [#3291](https://github.com/chocolatey/choco/issues/3291).
```
Comment (pauby) [inline input/en-us/choco/release-notes.md:58]:
```suggestion
- Fix - Unable to do non-elevated operations when `cacheLocation` is set to a restricted directory - see [#3291](https://github.com/chocolatey/choco/issues/3291).
```

**Review by pauby — CHANGES_REQUESTED** [review]

File: input/en-us/choco/release-notes.md [hunk 4]
```diff
@@ -46,6 +46,27 @@ This covers changes for the "chocolatey" and "chocolatey.lib" packages, which ar
 - Improve error message that is shown when defaultPushSource configuration value hasn't been set and no explicit source has been provided - see [#3280](https://github.com/chocolatey/choco/pull/3280).
 
 
+## 2.2.1 (August 3, 2023)
+
+> :choco-warning: **WARNING**
+>
+> Just prior to the public release of Chocolatey CLI 2.2.1 an issue was identified which meant that the release could not be completed.
+> Instead, work will be completed to correct this issue, and a new Chocolatey CLI 2.2.2 release will be completed.
```
Comment (pauby) [inline input/en-us/choco/release-notes.md:54]:
```suggestion
>  Chocolatey CLI 2.2.1 was removed from public availability due to a bug identified just prior to the release.
```

**Review by pauby — CHANGES_REQUESTED** [review]

**Review by pauby — APPROVED** [review]

**Pull request merged**